### ISSUE TITLE
Automatic ticket buying doesn't work; ticket buying doesn't work

### ISSUE THREAD

**Issue by vanjan** [decred/decrediton#3473]
I'm seeing the following errors in dcrwallet log:
```
2021-05-20 12:13:52.444 [ERR] VSPC: ticket 93a78312b8182e79ad7c2d607b3481dd06f388deb577efd6bddbfe0542dcdf81: reconcile payment: payfee: POST https://vsp.coinmine.pl/api/v3/payfee: Post "https://vsp.coinmine.pl/api/v3/payfee": write tcp 192.168.50.242:63206->172.67.188.60:443: write: can't assign requested address
2021-05-20 12:13:52.444 [ERR] VSPC: ticket e7c889bf059fd9502e36ac38b19286e2b1624b8ab0d7528b9853043fd33f18d6: reconcile payment: payfee: POST https://vsp.coinmine.pl/api/v3/payfee: Post "https://vsp.coinmine.pl/api/v3/payfee": http2: client connection force closed via ClientConn.Close

2021-05-20 12:22:51.714 [WRN] VSPC: Rescheduling status check for 2332ef698a312227e5d329f4c863506ff434f91965b67fc5d3279b83587115a1: POST https://vsp.coinmine.pl/api/v3/ticketstatus: Post "https://vsp.coinmine.pl/api/v3/ticketstatus": read tcp 172.33.12.100:63217->172.67.188.60:443: read: operation timed out
2021-05-20 12:22:51.714 [WRN] VSPC: Rescheduling status check for 93a78312b8182e79ad7c2d607b3481dd06f388deb577efd6bddbfe0542dcdf81: POST https://vsp.coinmine.pl/api/v3/ticketstatus: Post "https://vsp.coinmine.pl/api/v3/ticketstatus": read tcp 172.33.12.100:63217->172.67.188.60:443: read: operation timed out
2021-05-20 12:22:51.714 [WRN] VSPC: Rescheduling status check for 6d71f13e41a2289866db7e334ce289a21fec527354aa48c6ebbd2d67caebc707: POST https://vsp.coinmine.pl/api/v3/ticketstatus: Post "https://vsp.coinmine.pl/api/v3/ticketstatus": read tcp 172.33.12.100:63217->172.67.188.60:443: read: operation timed out

2021-05-20 12:23:51.175 [INF] GRPC: Unary method /walletrpc.WalletService/PurchaseTickets invoked by 127.0.0.1:58438
2021-05-20 12:23:51.278 [INF] WLLT: Reserved credits for 6 tickets: total fee: 0.00373249 DCR
2021-05-20 12:23:51.298 [INF] WLLT: Returning address (account=0 branch=1 child=22201)
2021-05-20 12:23:51.301 [INF] WLLT: Returning address (account=0 branch=1 child=22202)
2021-05-20 12:23:51.305 [INF] WLLT: Inserting unconfirmed transaction a93dfc4143f54a9569d1b1b83244890b860fed0e1af9c6b8d40f40096af6d75a
2021-05-20 12:23:51.334 [ERR] WLLT: Abandoning transaction a93dfc4143f54a9569d1b1b83244890b860fed0e1af9c6b8d40f40096af6d75a which failed to publish
2021-05-20 12:23:51.358 [ERR] GRPC: Unary method /walletrpc.WalletService/PurchaseTickets invoked by 127.0.0.1:58438 errored: wallet.PurchaseTicketsWithResponse:: individualSplit:: dcrd.PublishTransactions: rejected transaction a93dfc4143f54a9569d1b1b83244890b860fed0e1af9c6b8d40f40096af6d75a: orphan transaction a93dfc4143f54a9569d1b1b83244890b860fed0e1af9c6b8d40f40096af6d75a references outputs of unknown or fully-spent transaction a3f3cab22bd1529a1b66be3cac3ce9266efe1fdecabb592ad679cf67d23be1ed
2021-05-20 12:23:53.247 [INF] GRPC: Unary method /walletrpc.WalletService/GetPeerInfo invoked by 127.0.0.1:58438

2021-05-20 12:26:33.070 [INF] GRPC: Unary method /walletrpc.WalletService/PurchaseTickets invoked by 127.0.0.1:58438
2021-05-20 12:26:33.185 [INF] WLLT: Reserved credits for 1 tickets: total fee: 0.00373249 DCR
2021-05-20 12:26:33.204 [INF] WLLT: Returning address (account=0 branch=1 child=22203)
2021-05-20 12:26:33.207 [INF] WLLT: Returning address (account=0 branch=1 child=22204)
2021-05-20 12:26:33.209 [INF] WLLT: Inserting unconfirmed transaction 13ae14bf6e3c3f968742ba6dbd059850d09fb0e4e291f3c5549d9c338282721c
2021-05-20 12:26:33.230 [ERR] WLLT: Abandoning transaction 13ae14bf6e3c3f968742ba6dbd059850d09fb0e4e291f3c5549d9c338282721c which failed to publish
2021-05-20 12:26:33.253 [INF] GRPC: Unary method /walletrpc.WalletService/GetPeerInfo invoked by 127.0.0.1:58438
2021-05-20 12:26:33.255 [ERR] GRPC: Unary method /walletrpc.WalletService/PurchaseTickets invoked by 127.0.0.1:58438 errored: wallet.PurchaseTicketsWithResponse:: individualSplit:: dcrd.PublishTransactions: rejected transaction 13ae14bf6e3c3f968742ba6dbd059850d09fb0e4e291f3c5549d9c338282721c: orphan transaction 13ae14bf6e3c3f968742ba6dbd059850d09fb0e4e291f3c5549d9c338282721c references outputs of unknown or fully-spent transaction 0335d180fca929d7962b97678ac4c993b993ae148b41a3c6a6693105d8e05dc9
2021-05-20 12:26:38.254 [INF] GRPC: Unary method /walletrpc.WalletService/ValidateAddress invoked by 127.0.0.1:58438
```

Using v1.6.3

**Comment by lolandhold** [comment]
```
2021-05-20 12:23:51.305 [INF] WLLT: Inserting unconfirmed transaction a93dfc4143f54a9569d1b1b83244890b860fed0e1af9c6b8d40f40096af6d75a
2021-05-20 12:23:51.334 [ERR] WLLT: Abandoning transaction a93dfc4143f54a9569d1b1b83244890b860fed0e1af9c6b8d40f40096af6d75a which failed to publish
```
I see this too.

**Comment by vanjan** [comment]
Seems to be working upon restart. What I did differently this time is I waited patiently after the initial screen asking me to process managed tickets. Previously I kept entering my password multiple times, because after you do it the first time nothing seemingly happens, but actually something is happening in the background and the problem is the UI gives no feedback to that effect, making you keep on retrying to do the same thing, which seems to be causing issues down the line.
